docs/rpc_nodes.md

```diff
@@ -23,7 +23,7 @@ values={[
 
 | Provider         | Net          | URL                                      | Header                                                                          |
 |------------------|--------------|------------------------------------------|---------------------------------------------------------------------------------|
-| ECAD Labs        | Mainnet      | https://mainnet.api.tez.ie               | [Check](https://mainnet.api.tez.ie/chains/main/blocks/head/header)              |
+| ECAD Labs        | Mainnet      | https://mainnet.ecadinfra.com               | [Check](https://mainnet.ecadinfra.com/chains/main/blocks/head/header)              |
```

Comment:
mainnet header link isn't working https://mainnet.ecadinfra.com/chains/main/blocks/head/header

Reply:
That was an infrastructure issue that has since been resolved. Thanks for picking that up @hui-an-yang